src/components/InjectedComponents/TypedMessageRenderer.tsx

```diff
@@ -62,6 +70,19 @@ export const DefaultTypedMessageTextRenderer = React.memo(function DefaultTypedM
     )
 })
 
+export const DefaultTypedMessageImageRenderer = React.memo(function DefaultTypedMessageImageRenderer(
+    props: TypedMessageRendererProps<TypedMessageImage>,
+) {
+    console.log(props.message.content)
+    return renderWithMetadata(
+        props,
+        <Typography variant="body1" style={{ lineBreak: 'anywhere' }} data-testid="text_payload">
+            {/* <RenderText text={props.message.content}></RenderText> */}
+            <img style={{ width: '100%' }} src={`data:image/png;base64,${props.message.content}`} />
```

Comment:
Notice: data url might be blocked to render. please use a canvas to draw the image

Reply:
Should be done by another PR.